src/urbit/mar/chess/game/archived.hoon

```diff
@@ -18,7 +18,10 @@
         ['white' [%s (player-string:chess white.game)]]
         ['black' [%s (player-string:chess black.game)]]
         ['result' [%s ?~(result.game '' u.result.game)]]
+        ::  default values. updated thru [%archived-games ~] sub.
         ['moves' [%a ~]]
+        ['archived' [%b %.y]]
```

Comment:
nit: I'd move this to the above section, as I don't consider this to be a default value